backend-tests/tests/test_decommission.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import pytest
+import random
+import time
+import logging
+import re
+
+from testutils.common import mongo, clean_mongo
```

Comment:
Unused import mongo

Reply:
clean_mongo is using this fixture, that's why we need to import it here

Reply:
Thanks for clarifying